sunkit_pyvista/plotter.py

```diff
@@ -45,16 +45,16 @@ class SunpyPlotter:
         Stores a reference to all the plotted meshes in a dictionary.
     """
 
-    def __init__(self, *, coordinate_frame=None, obstime=None, **kwargs):
+    def __init__(self, *args, coordinate_frame=None, obstime=None, **kwargs):
+        super().__init__(*args, **kwargs)
         if coordinate_frame is not None and obstime is not None:
             msg = "Only coordinate_frame or obstime can be specified, not both."
             raise ValueError(msg)
         if coordinate_frame is None:
             coordinate_frame = HeliocentricInertial(obstime=obstime)
         self._coordinate_frame = coordinate_frame
-        self._plotter = pv.Plotter(**kwargs)
-        self.camera = self._plotter.camera
         self.all_meshes = {}
+        self._initialized = True
```

Comment:
What's this do?

Reply:
Err, not a clue but its the base plotter was setup upstream and I seemed to need it.

Reply:
Turns out I am wrong. This was not needed. I'm just stupid.